### PR TITLE
[CodeStyle][task 1] enable Ruff UP032 rule with some comments

### PR DESCRIPTION
**Pull request by Liyulingyue**
### PR types

Others
### PR changes

Others
### Description

- https://github.com/PaddlePaddle/Paddle/issues/57367 task1
- https://github.com/PaddlePaddle/Paddle/pull/57409

俺觉得这样写很合理 https://github.com/PaddlePaddle/Paddle/pull/57409/files/de30b7630fa13d9bcbda3b8ef5b919d16df975d0#diff-f8bd29210e2434bd3a5668c7755e422287d538f259830ce6d3e3ef99ab7d2a63

### CODE REVIEW

File: python/paddle/tensor/to_string.py [hunk 1]
```diff
@@ -126,15 +126,11 @@ def _format_item(np_var, max_width=0, signed=False):
...
-                np_var
-            )
+            item_str = f'{np_var:.{DEFAULT_PRINT_OPTIONS.precision}e}'
         elif np.ceil(np_var) == np_var:
             item_str = f'{np_var:.0f}.'
         else:
-            item_str = f'{{:.{DEFAULT_PRINT_OPTIONS.precision}f}}'.format(
-                np_var
-            )
+            item_str = f'{np_var:.{DEFAULT_PRINT_OPTIONS.precision}f}'
```
Comment (gouzil) [inline python/paddle/tensor/to_string.py:133]: 帮你验证一下
```bash
>>> DEFAULT_PRINT_OPTIONS="2"
>>> np_var=1.23324
>>> f'{np_var:.{DEFAULT_PRINT_OPTIONS}f}'
'1.23'
>>> f'{np_var:.{DEFAULT_PRINT_OPTIONS}e}'
'1.23e+00'
```

File: test/legacy_test/test_inplace.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def test_backward_error(self):
             loss = paddle.nn.functional.relu(var_c + var_d)
             with self.assertRaisesRegex(
                 RuntimeError,
-                f"received tensor_version:{1} != wrapper_version_snapshot:{0}",
+                "received tensor_version:1 != wrapper_version_snapshot:0",
```
Comment (gouzil) [inline test/legacy_test/test_inplace.py:59]: 这个文件下还有 10 处未修改的，可以使用`\{\d+\}`正则搜索

**Review by gouzil — APPROVED** [review]
LGTM，@SigureMo

**Review by SigureMo — APPROVED** [review]
LGTMeow 🐾

**Pull request merged**